icone_arrow_boards/process_icone_logs.py

```diff
@@ -0,0 +1,56 @@
+########################################################################################################################
+# This script processes the log files created by the log_icone.py script and creates a JSON file with the start and end times of the log as well as the segments of the log with the ID, start and end times, states, and coordinates of each segment.
+########################################################################################################################
+
+import json
+
+# (2022[0-9]{4}-[0-9]{6})
```

Comment:
Seems to be a few random comments fin here, should we clean that up?

Reply:
Resolved in PR #51